tests/Acme81/Attribute/ParameterA.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Acme81\Attribute;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_PARAMETER)]
+class ParameterA
+{
+    public string $label;
+    public function __construct(
+        string $label = ''
+    ) {
+        $this->label = $label;
+    }
```

Comment:
We can use promoted properties here:

```suggestion
    public function __construct(
        public string $label = ''
    ) {
    }
```